docs/docs/components/README.md

```diff
@@ -0,0 +1,108 @@
+# Overview
+
+- [What are Components?](#what-are-components-)
```

Comment:
You can use the `[[toc]]` shorthand to include a table of contents, and it'll be generated and formatted with sub-bullets based on the header heirarchy. [See the VuePress docs](https://v1.vuepress.vuejs.org/guide/markdown.html#table-of-contents). Then if you change the title / order of a section, you don't have to modify the ToC manually.

Reply:
done. thanks!